rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -996,6 +996,54 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
+        eventLogger.logCommitFederationFailure(rskExecutionBlock, proposedFederation);
+        logger.warn("[processValidationFailure] Proposed federation validation failed so svp values will be cleared.");
+        clearSvpValues();
+    }
+
+    private void clearSvpValues() {
+        federationSupport.clearProposedFederation();
+
+        String methodName = "[clearSvpValues]";
+
+        Optional<Sha256Hash> svpFundTxHashUnsigned = provider.getSvpFundTxHashUnsigned();
+        if (svpFundTxHashUnsigned.isPresent()) {
+            logger.warn("{} Fund tx {} was never registered.", methodName, svpFundTxHashUnsigned.get());
+            provider.setSvpFundTxHashUnsigned(null);
+            return;
+        }
```

Comment:
In this scenario we can avoid the isPresent / get pattern, it can be nicely be simplified as:
```java
provider.getSvpFundTxHashUnsigned().ifPresent(txHash -> {
    logger.warn("{} Fund tx {} was never registered.", methodName, txHash);
    provider.setSvpFundTxHashUnsigned(null);
});
```
This pattern can be replicated for all the other sections of this method

Reply:
nice, thanks :)

Reply:
updated